### ISSUE TITLE
Sample xxxx does not contain the 2 genes when extracts the region surrounding a given pair of genes

### ISSUE THREAD

**Issue by drxilu** [gtonkinhill/panaroo#179]
Hello there, I am using `extractGeneRegionFromGMLGenes.py` script to extract genes between certrain regions from panaroo .gml output.
Followed instructions from https://github.com/gtonkinhill/panaroo/blob/master/scripts/extractGeneRegionFromGMLGenes.py, and ran `python extractGeneRegionFromGMLGenes.py -n .gmlFile -d gene_data.csv -g 2xGenesOfInterest -l LengthOfRegionToExtract -t AssemblyFileTranslation -o OutFile`
I ran commands like:
`python extractGeneRegionFromGMLGenes.py -n panaroo_final_graph.gml -d panaroo_gene_data.csv -g geneName1 geneName2 -l 50000 -t gff_list.txt -o extract_cps`

But I got error messages like,
```
Traceback (most recent call last):

  File "extractGeneRegionFromGMLGenes.py", line 211, in <module>
   
  gffFiles[sample], contigNames[sample][0],

KeyError: '86'
```
And output like:
```
Sample 0 does not contain the 2 genes

Sample 1 does not contain the 2 genes

Sample 2 does not contain the 2 genes
.....

Sample 84 does not contain the 2 genes

Sample 85 does not contain the 2 genes
```
May I gently check if I miss anything? Thank you!

**Comment by gtonkinhill** [comment]
Hi,

This script is not used very frequently and thus does not provide very detailed usage information. The current version requires that the GFF files are provided as separate entries on the command line and not as a list in a file.

I have now pushed a small update that allows for a list in a file to be provided instead. This will be available in the next release but can be accessed now by reinstalling Panaroo using

```
pip install git+https://github.com/gtonkinhill/panaroo@devel
```

Alternatively, you could look at using [Magphi](https://github.com/milnus/Magphi) which solves a similar problem.

**Comment by drxilu** [comment]
Thank you for your reply, and let me know these details when using GFF files as separate entries.

Looking forward to seeing the new update when it is available. Meanwhile, I will also try Magphi as suggested if needed.

Thanks again for your help!

**Comment by gtonkinhill** [comment]
This update should now be available in the latest release v1.3.0
